### PR TITLE
Check valid configuration before submitting

### PR DESCRIPTION
**Pull request by zenine07**
#  #

## 🗣 Description ##

Make request to backend to check whether configuration is valid.

## 💭 Motivation and context ##

There are sets of parameters defined at https://www.mailgun.com/blog/domain-warmup-reputation-stretch-before-you-send/ for sending email and slowly ramping up emails. This makes a request to the backend to verify the subscription remains within those parameters.

## 🧪 Testing ##

Tested significantly locally. Also created pytests on the backend for additional tests.

## ✅ Checklist ##

* [x] This PR has an informative and human-readable title.
* [x] Changes are limited to a single goal - _eschew scope creep!_
* [x] _All_ future TODOs are captured in issues, which are referenced
      in code comments.
* [x] All relevant type-of-change labels have been added.
* [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
* [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
* [x] Tests have been added and/or modified to cover the changes in this PR.
* [x] All new and existing tests pass.

### CODE REVIEW

**Comment by jsf9k** [comment]
@zenine07, in INL's PRs you should list at least one person as an assignee.  Assignees are folks who can make the final decision as to whether a PR is ready to be merged and actually click the merge button.  (Normally we just have a single assignee, which is the person who created the PR.)

**Review by jsf9k — APPROVED** [review]

**Review by itsmostafa — APPROVED** [review]

**Pull request merged**
